assets/sass/vendor/_mdc-dialog.scss

```diff
@@ -2,24 +2,17 @@
 .googlesitekit-plugin .mdc-dialog {
 	z-index: 10000;
 
-	%label-small {
-		font-size: $fs-body-sm;
-		letter-spacing: 0.2px;
-		line-height: $lh-label-sm;
-	}
-
 	.mdc-dialog__surface {
-		border-radius: 8px;
-		box-shadow: 0 10px 40px 0 rgba(#000, 0.35);
-		padding: $grid-gap-desktop + 12px $grid-gap-desktop * 2 - 10; // 36px 38px.
+		border-radius: $br-xs;
+		box-shadow: 0 10px 40px 0 rgba($c-black, 0.35);
+		max-width: 76vw;
+		padding: $grid-gap-desktop;
 
 		@media (min-width: $bp-tablet) {
-			max-width: 478px;
-		}
-
-		@media (max-width: $bp-mobileOnly) {
-			max-width: 76vw;
-			padding: $grid-gap-desktop;
+			/* Differs from what is seen in figma design - 478px,
+			after disscusion with Sigal https://10up.slack.com/archives/C01CA828ZPZ/p1707129589297719 */
+			max-width: 590px;
```

Comment:
I have asked a question for confirmation [here](https://10up.slack.com/archives/C01CA828ZPZ/p1707207601668489?thread_ts=1707129589.297719&cid=C01CA828ZPZ) as it looks a bit cramped in the minimum viewport (600px), so please let's wait for a response there, thanks!

![image](https://github.com/google/site-kit-wp/assets/20284937/cd3d9820-38e9-4b5f-931b-2b07e67c963f)

Reply:
Reverting to 478px per Sigal's reply

Reply:
According to our conversation, let's use 478px for tablet and above, but 590px for desktop & above. Thanks!